lesson26/src/js/login.js

```diff
@@ -0,0 +1,69 @@
+const userIdOfInput = document.querySelector(".js-form-userid");
+const passwordOfInput = document.querySelector(".js-form-password");
+const formElements = [userIdOfInput,passwordOfInput];
+const submitButton = document.getElementById("js-submit-button");
+
+const isNotEmptyOfInput = target => target.value.trim() !== "";
+const isNotEmptyOfAllInput = () => {
+    let result = false;
+    formElements.forEach(element => {
+        element.value.trim() !== "" ? result = true : result = false;
+    })
+    return result;
```

Comment:
IMO
You can also write like this

```suggestion
    return formElements.every((element) => isNotEmptyOfInput(element))
```

or

```suggestion
    return formElements.some((element) => isNotEmptyOfInput(element))
```

``` js
const isNotEmptyOfInput = target => target.value.trim() === "";
```

Reply:
Thank you for your review!
I was able to find out about every() and some().Thank you!

This time, I added and modified password validation as well as issue 25 in Chihiro-san's review.
Therefore, we did not use the isNotEmptyOfAllInput() function,
but changed the method to give an INVALID class if the validation does not pass.
[c99ad95](https://github.com/haru-programming/JavaScript-lessons/pull/30/commits/c99ad95537d0b7227bb4b17a019dc94e9e9da9ef)

but,
I noticed that I don't have to invert the result by changing `isNotEmptyOfInput -> isEmptyOfInput`.
This has been corrected.
[3041930](https://github.com/haru-programming/JavaScript-lessons/pull/30/commits/30419303a885805df76e2527916120f309c2afce)
[9c1ab68](https://github.com/haru-programming/JavaScript-lessons/pull/30/commits/9c1ab688b9d15a4b72a1f0632c816b09d5bd8f1f)(Sorry..I mistake in 
operator)